snykTask/src/task-args.ts

```diff
@@ -150,6 +150,19 @@ class TaskArgs {
       const errorMsg = `If set, codeSeverityThreshold must be one from [${taskTestTypeThreshold}] (case insensitive). If not set, the default is '${Severity.LOW}'.`;
       throw new Error(errorMsg);
     }
+
+    // code output json dependent on tested severity threshold but does not describe issues by its severity.
+    if (
+      taskTestType === TestType.CODE &&
+      ((this.codeSeverityThreshold &&
+        this.failOnThreshold &&
+        this.codeSeverityThreshold.toLowerCase() !==
+          this.failOnThreshold.toLowerCase()) ||
+        (!this.codeSeverityThreshold && this.failOnThreshold !== Severity.LOW))
```

Comment:
question: it looks like `codeSeverityThreshold` is defaulted to `Severity.LOW` and `failOnThreshold` is defaulted to an empty string, both of which makes the presence checks unnecessary.

I also noted that `failOnThreshold` is defaulted to an empty string in code but the README states it's default value is `low`; is this correct?

Reply:
the README refers from a user perspective that the Azure Task UI editor in which the combo dropdown for `codeSeverityThreshold`  is preset to `low`.